solidity/contracts/Gravity.sol

```diff
@@ -63,7 +65,7 @@ contract Gravity is ReentrancyGuard {
 	CudosAccessControls public cudosAccessControls;
 
 	mapping(address => bool) public whitelisted;
-
+	mapping(address => bool) public supportedToCosmosTokens;
```

Comment:
shouldn't this be Cudos instead of cosmos?

Reply:
This was named so because the function is named `sendToCosmos`. Should I still rename the variable?